keyboards/work_louder/rgb_functions.c

```diff
@@ -17,13 +17,160 @@
 #include QMK_KEYBOARD_H
 #include "rgb_functions.h"
 
-#if defined(RGBLIGHT_ENABLE) && defined(RGB_MATRIX_EANBLE)
-#    undef RGB_DI_PIN
-#    define RGBLIGHT_DI_PIN
-#    include "ws2812.c"
+#ifdef RGBLIGHT_ENABLE
```

Comment:
???

Reply:
It uses ws2812 on 2 different pins. And one for underglow and one for rgb matrix.

I tried a hack that didn't really work

Reply:
/cry

Reply:
Tell me about it :(

board was already designed.  I'm just happy that it works fine, with no apparent issues